Parsers/ASimAuthentication/ProductParsers/ASimAuthenticationPostgreSQL.yaml

```diff
@@ -108,18 +120,22 @@ ParserQuery: |
     EventVendor = 'PostgreSQL'
     , EventProduct = 'PostgreSQL'
     , EventCount = int(1)
+    , EventSchema = 'Authentication'
     , EventSchemaVersion = '0.1.1'
     , EventResult = 'Success'
     , EventStartTime = TimeGenerated
     , EventEndTime = TimeGenerated
-    , EventType = 'Authorization'
+    , EventType = 'Logoff'
+    , DvcHostname = Computer
+    , DvcIpAddr = extract(@'\d{1,3}\.\d{1.3}\.\d{1,3}\.\d{1,3}', 1, Computer)
     , TargetUsernameType = 'Simple'
     , TargetUsername = extract(@'user=(.*?)\sdatabase', 1, RawData)
-    , SrcIPAddr = extract(@'host=\[?(.*?)\]?', 1, RawData)
-    , EventMessage = 'User session closed'
+    , SrcIpAddr = extract(@'host=\[?(.*?)\]?', 1, RawData)
+    , EventResultDetails = 'User session closed'
   // **** Aliases
   | extend
           User=TargetUsername
         , Dvc=Computer
     };
-  union isfuzzy=true PostgreSQLSignInAuthorized(disabled), PostgreSQLAuthFailure1(disabled), PostgreSQLAuthFailure2(disabled), PostgreSQLAuthFailure3(disabled), PostgreSQLDisconnect(disabled)
\ No newline at end of file
+  union isfuzzy=true PostgreSQLSignInAuthorized(disabled), PostgreSQLAuthFailure1(disabled), PostgreSQLAuthFailure2(disabled), PostgreSQLAuthFailure3(disabled), PostgreSQLDisconnect(disabled)
+  | project-away Computer, MG, ManagementGroupName, RawData, SourceSystem, TenantId
```

Comment:
**project-away** in each branch of the union rather than after the union. It has dramatic effect on memory use.

Reply:
fixed